pkg/kubelet/server/server.go

```diff
@@ -155,6 +156,9 @@ type HostInterface interface {
 	ResyncInterval() time.Duration
 	GetHostname() string
 	LatestLoopEntryTime() time.Time
+
+	// Used by fs_resource_analyzer
```

Comment:
(nit) I don't like "Used by / for" comments since they tend to become out of date (so code search is more reliable anyway).